rex-engine/src/codec.rs

```diff
@@ -344,6 +384,13 @@ impl Decode for String {
     }
 }
 
+impl Decode for ()
+{
+    fn try_decode(_v: &Expr) -> Result<Self, Error> {
+        Ok(())
+    }
```

Comment:
This should probably explicitly check that the expression is an empty tuple expression, and error if it's not.